.flake8

```
@@ -1,5 +1,33 @@
 [flake8]
-ignore = E203, E266, E501, W503, F403, F401
+ignore =
+    E203
+    E266
+    E501
+    W503
+    F403
+    F401
+    D401  # Imperative mood -- doesn't work for cached-property.
+    N802  # TODO: remove this (functino name should be lower-case)
```

those darn functinos always getting in trouble
```suggestion
    N802  # TODO: remove this (function name should be lower-case)
```